tests/director/external-services-mock-integration/fixtures_request.go

```diff
@@ -0,0 +1,83 @@
+package external_services_mock_integration
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/kyma-incubator/compass/components/gateway/pkg/auditlog/model"
+	gcli "github.com/machinebox/graphql"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	auditlogTokenEndpoint  = "audit-log/v2/oauth/token"
+	auditlogSearchEndpoint = "audit-log/v2/configuration-changes/search"
+)
+
+type token struct {
+	AccessToken string `json:"access_token"`
+	TokenType   string `json:"token_type"`
+	ExpiresIn   int    `json:"expires_in"`
+}
+
+// Application
+func fixRegisterApplicationRequest(applicationInGQL string) *gcli.Request {
+	return gcli.NewRequest(
+		fmt.Sprintf(`mutation {
+			result: registerApplication(in: %s) {
+					%s
+				}
+			}`,
+			applicationInGQL, tc.gqlFieldsProvider.ForApplication()))
+}
+
+func fixDeleteApplicationRequest(t *testing.T, id string) *gcli.Request {
+	return gcli.NewRequest(
+		fmt.Sprintf(`mutation {
+		unregisterApplication(id: "%s") {
+			%s
+		}	
+	}`, id, tc.gqlFieldsProvider.ForApplication()))
+}
+
+// External services mock
+func getAuditlogMockToken(t *testing.T, client *http.Client, baseURL, clientID, clientSecret string) token {
```

Comment:
I suggest not passing clientID and secret if we don't validate them on the audit log mock side.